irc/irc.php

```diff
@@ -24,7 +26,7 @@ function irc_uninstall() {
 }
 
 
-function irc_addon_settings(&$a,&$s) {
+function irc_addon_settings(App $a, $s) {
```

Comment:
The `&` operator of the `$s` parameter enables this function to change the hook data. If you remove it this function has no effect whatsoever.